java/src/jmri/jmrit/cabsignals/CabSignalPane.java

```diff
@@ -47,10 +39,11 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Pane for monitoring and configuring a MERG CBUS Command Station
- * Created with Notepad++
+ * Pane for sending Cab Signal data via block lookup
  * @author Steve Young Copyright (C) 2018
- * @since 4.13.4
```

Comment:
I would go ahead and leave these at 4.13.4.  That is when the original functionality was introduced, even though the generic tool came later.

Reply:
Thanks, makes sense.